config.json

```diff
@@ -8,23 +8,58 @@
     "representer": false,
     "analyzer": false
   },
-  "blurb": "TODO: add blurb",
+  "blurb": "Roc is a fast, friendly, functional language.",
   "version": 3,
   "online_editor": {
     "indent_style": "space",
-    "indent_size": 4
+    "indent_size": 4,
+    "highlightjs_language": "roc"
```

Comment:
It looks like this doesn't work by default, as Roc is not listed in the supported languages: https://github.com/highlightjs/highlight.js/blob/main/SUPPORTED_LANGUAGES.md Is there a highlightjs plugin for Roc published on NPM?

Reply:
No, I don't think so

Reply:
Then I would either:

- revert, or 
- change to a language with similar syntax